src/linagora.esn.calendar/app/components/modals/calendar-secret-address-confirmation/calendar-secret-address-confirmation-modal.service.js

```diff
@@ -0,0 +1,21 @@
+(function(angular) {
+  'use strict';
+
+  angular.module('esn.calendar')
+    .factory('calCalendarSecretAddressConfirmationModalService', calCalendarSecretAddressConfirmationModalService);
```

Comment:
Please make sure you test this in production mode.

Reply:
yes , it is working